fortinet-fortindr-cloud/info.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fortinet-fortindr-cloud",
   "label": "Fortinet FortiNDR Cloud",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
@cs-shubham-morankar, If only a minor bug fix is included in this release, then we can release a minor version of the connector. We follow this same process for all connectors.